transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/ExpiryDateWidget.java

```diff
@@ -186,6 +186,7 @@ public void afterTextChanged(Editable s) {
             editText.setText(mExpireDateUtils.convertDateFromServerFormat(mDefaultValue));
 
             editText.setOnKeyListener(new DefaultKeyListener(mDefaultFocusView, editText));
+            editText.addTextChangedListener(new TextChangeListener());
```

Comment:
ExpiryDateWidget already has a `TextChangedListener` why assign a new one?

Reply:
https://developer.android.com/reference/android/view/View.OnKeyListener
OnKeyListener not triggered in some keyboard, google recommended to use textchange listener for text changes.

Reply:
<img width="1059" alt="Screenshot 2021-02-26 at 2 23 10 PM" src="https://user-images.githubusercontent.com/71114945/109278176-3ce6db80-783e-11eb-87f2-f2a8809a8a28.png">

Reply:
fixed